### PR TITLE
Move AdminBundleIT to a separate profile

### PR DESCRIPTION
**Pull request by pgesek**
The IT was turned into an FT that will run
only in the FT profile. For now this is acceptable.

### CODE REVIEW

**Comment by pgesek** [comment]
http://ci.motechproject.org/job/Motech-Admin-FT/ will build these tests daily

File: modules/admin/pom.xml [hunk 1]
```diff
@@ -363,4 +359,46 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>FT</id>
```
Comment (sebbrudzinski) [inline modules/admin/pom.xml]: This profile should also handle the copying and cleaning of the configuration on its own.

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
@pgesek OK to merge

**Pull request merged**